packages/create-onchain/src/cli.ts

```diff
@@ -192,6 +475,85 @@ async function init() {
   console.log(' - npm run dev');
 }
 
+export function getArgs() {
+  const options = {
+    isHelp: false,
+    isVersion: false,
+    isGenerate: false,
+    isMiniKit: false,
+  };
+
+  // find any argument with -- or -
+  const arg = process.argv.find(
+    (arg) => arg.startsWith('--') || arg.startsWith('-'),
+  );
+  switch (arg) {
+    case '-h':
+    case '--help':
+      options.isHelp = true;
+      break;
+    case '-v':
+    case '--version':
+      options.isVersion = true;
+      break;
+    case '-g':
+    case '--generate':
```

Comment:
issue: IMO, everything create-onchain does is some type of generation, I think we should name this something more descriptive. --frames-manifest, --mini-manifest, --manifest if we really need to keep those short